From b576918bd2c23a6359ba07f88a84b09e94497bbb Mon Sep 17 00:00:00 2001
From: Petr Vobornik <pvoborni@redhat.com>
Date: Mon, 9 Sep 2013 10:07:09 +0200
Subject: [PATCH] Fix RUV search scope in ipa-replica-manage

The search had an incorrect scope and therefore it didn't find any RUV.

This issue prevented removing of replica.

https://fedorahosted.org/freeipa/ticket/3876
---
 install/tools/ipa-replica-manage | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install/tools/ipa-replica-manage b/install/tools/ipa-replica-manage
index d7e317c5a09d24c63b1a890c98c0a0c621050ae2..9e5cbf5218365fb8dee488ed794f915e7db5eab7 100755
--- a/install/tools/ipa-replica-manage
+++ b/install/tools/ipa-replica-manage
@@ -317,7 +317,7 @@ def get_ruv(realm, host, dirman_passwd):
 
     search_filter = '(&(nsuniqueid=ffffffff-ffffffff-ffffffff-ffffffff)(objectclass=nstombstone))'
     try:
-        entries = thisrepl.conn.search_s(api.env.basedn, ldap.SCOPE_ONELEVEL,
+        entries = thisrepl.conn.search_s(api.env.basedn, ldap.SCOPE_SUBTREE,
             search_filter, ['nsds50ruv'])
     except ldap.NO_SUCH_OBJECT:
         print "No RUV records found."
-- 
1.8.3.1

